docs/pages/User/Installation/aks.md

```diff
@@ -0,0 +1,161 @@
+---
+title: AKS
+weight: 1
+---
+
+This guide will show you how to install Liqo on your AKS cluster
+
+### Requirements
+
+* an [Azure account](https://azure.microsoft.com/auth/signin/?loginProvider=Microsoft&redirectUri=%2Fen-us%2F)
+
+### Steps
+
+* [Create an AKS Cluster](#create-an-aks-cluster)
+* [Enable required features](#enable-required-features)
+* [Deploy Liqo](#deploy-liqo)
+* [Check that Liqo is Running](#check-that-liqo-is-running)
+
+## Create an AKS Cluster
+
+### Access Azure Portal
+
+Access in the Azure [Portal](https://portal.azure.com/) to the [Kubernetes Service](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.ContainerService%2FmanagedClusters).
+
+<!-- TODO: change all paths -->
+![](/images/install/aks/01.png)
+
+### Create a new Cluster
+
+Click on `Add` > `Add Kuberntes cluster` to create a new cluster. A new panel wil appear.
+
+Select the desired `Subscription` and `Resource Group`, choose a name, a region and an availability zone to assign to
+your cluster.
+
+__NOTE__: Liqo only supports a `Kubernetes version` >= 1.19.0
+
+Liqo does not require any other configurations to the cluster. You can click on the `Review + create` button.
+
+![](/images/install/aks/02.png)
+
+When the validation is passed, click on the `Create` button.
+
+![](/images/install/aks/03.png)
+
+Azure will take some minutes to deploy your cluster.
+
+## Enable required features
+
+### Enable HTTP Application Routing
+
+When your cluster has been completely deployed, you have to enable an ingress controller to make the Liqo Auth Service
+accessible from the external world.
+
+Azure has a built-in plugin that enable this feature, although it is not recommended for use in production, called
+[HTTP application routing](https://docs.microsoft.com/en-US/azure/aks/http-application-routing).
+
+To enable it on your cluster dashboard go in `Settings` > `Networking` and make sure that the _Enable HTTP application
+routing_ checkbox is enabled and click on the `Save` button.
+
+![](/images/install/aks/04.png)
+
+Azure will take some minutes to deploy required components and enable the required services.
+
+When the operation is completed you will see a new [DNS zone](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Network%2FdnsZones)
+in the Azure Portal.
+
+![](/images/install/aks/05.png)
+
+Congratulations! Your AKS cluster is now ready to run Liqo!
+
+## Deploy Liqo
+
+### One Line installer
+
+#### Export Variables
+
+The Liqo one line installer needs some environment variables to know where the Liqo components are accessible from the
+external world and how the local Kubernetes installation is configured.
+
+In particular we have to export the following environment variables:
```

Comment:
I really liked how the table is rendered in the very next section. Can we use a table also for the parameters listed here? What do you think?

Reply:
Yes sure, I can add it